server/config/mongodb_config.py

```diff
@@ -23,7 +23,7 @@ def load_env_file():
 class MongoConfig(object):
     """Base configuration class."""
     load_env_file()
-    MONGODB_HOST = os.getenv('MONGODB_HOST', 'mongodb')
+    MONGODB_HOST = os.getenv('MONGODB_HOST', 'mongod')
```

Comment:
When connecting to Mongodb Atlas, the host needs to be 'mongodb'. You should undo this change.